core/blockstm/executor.go

```diff
@@ -0,0 +1,241 @@
+package blockstm
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/log"
+)
+
+type ExecResult struct {
+	err      error
+	ver      Version
+	txIn     TxnInput
+	txOut    TxnOutput
+	txAllOut TxnOutput
+}
+
+type ExecTask interface {
+	Execute(mvh *MVHashMap, incarnation int) error
```

Comment:
Since we can't perfectly abstract MVHashmap from statedb, changed the `Execute` interface a bit by replacing the read/write interface with MVHashmap directly.